<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(165, 165, 165);
color: rgb(255, 255, 255);
background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="detect_button">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Detect</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/radar.png</normaloff>code/icons/radar.png</iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generate_report">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">radius {
	10px;
}</string>
        </property>
        <property name="text">
         <string>GR Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="profile">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/user.png</normaloff>code/icons/user.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>8</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Image Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="prev_button">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nex_button">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="insert_new">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/insert.png</normaloff>code/icons/insert.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_folder">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/folder.png</normaloff>code/icons/folder.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/save.png</normaloff>code/icons/save.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>C:/Users/matab/OneDrive/Pictures/PREMIERE! 2021 AUDI RSQ8-R 740HP - THE NEW MONSTER-SUV FROM ABT SPORTSLINE IN DETAIL.jpeg</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="zoom_in">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/zoom-in.png</normaloff>code/icons/zoom-in.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoom_out">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/zoom-out.png</normaloff>code/icons/zoom-out.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotate">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/brightness.png</normaloff>code/icons/brightness.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="brightness_up">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/rotate_image.png</normaloff>code/icons/rotate_image.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="brightness_down">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/edit-image.png</normaloff>code/icons/edit-image.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="split_button">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Split</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/split-screen.png</normaloff>code/icons/split-screen.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drawing_tools">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/expand.png</normaloff>code/icons/expand.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lock_button">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/text.png</normaloff>code/icons/text.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_button">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Lock</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/padlock.png</normaloff>code/icons/padlock.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drag_button">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>info</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>code/icons/info.png</normaloff>code/icons/info.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionShare"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBCDS_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_BCDS"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Image">
   <property name="icon">
    <iconset>
     <normaloff>code/icons/insert.png</normaloff>code/icons/insert.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset>
     <normaloff>code/icons/folder.png</normaloff>code/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>code/icons/save.png</normaloff>code/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="icon">
    <iconset>
     <normaloff>code/icons/share.png</normaloff>code/icons/share.png</iconset>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionBCDS_Help">
   <property name="text">
    <string>BCDS Help</string>
   </property>
  </action>
  <action name="actionAbout_BCDS">
   <property name="text">
    <string>About BCDS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
